src/main/ruby/core/truffle/interop.rb

```diff
@@ -34,6 +34,13 @@ def self.object_has_keys(object)
       end
     end
 
+    def self.keys(object, internal = false)
+      keys = Truffle.invoke_primitive :interop_send_keys, object, internal
+      Truffle::Interop.enumerable(keys).map { |key|
```

Comment:
I think we've settled on multi-line blocks using `do...end`.

Reply:
It feels weird for `map` but I can change.

Reply:
There is a `map` literally just below with `{}` and then a `each` with `do/end` following the return value rule. Do you think it's important to use `do/end` for map?

Reply:
I made it one line and removed the redundant `Truffle::Interop` prefixes :)